Cохранение и удаление записи в блоке рабочий стаж.

ageeva-olga opened the pull request.

ageeva-olga committed:
Добавила сохранение и удаление записи в блоке рабочий стаж. Без таблицы.

felun commented on MyCV/MyCV/DAL/WorkExperienceRepository.cs:
```diff
@@ -0,0 +1,67 @@
+            {
+                var sqlite_cmd = sqlite_conn.CreateCommand();
+                sqlite_cmd.CommandText =
+                    "SELECT Id, Begin, End, WorkName, PositionName FROM Education";
```
Почему репозиторий WorkExperience, а табличка Education?

ageeva-olga committed:
Исправила ошибки:
в контроллере,
добавила таблицу,
запрос в репозитории Work Experience,
модель Work Experience

ageeva-olga committed:
Merge branch 'master' into feature/PersonalInfoDatabaseIntegration
